test/dialect/postgresql/test_reflection.py

```diff
@@ -932,6 +932,49 @@ def go():
             ],
         )
 
+    @testing.provide_metadata
+    def test_index_reflection_partial(self):
+        """Reflect the filter defintion on partial indexes
+        """
+
+        metadata = self.metadata
+
+        t1 = Table(
+            "table1",
+            metadata,
+            Column("id", Integer, primary_key=True),
+            Column("name", String(20)),
+            Column("x", Integer),
+        )
+        metadata.create_all()
+        with testing.db.connect().execution_options(autocommit=True) as conn:
```

Comment:
**mike bayer** (zzzeek) wrote:

use connection, don't need autocommit=True as that's deprecated anyway

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142

Reply:
**Ramon Williams** (RamonWill) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142